content/docs/deployment/helm/drivers/installation/powerflex.md

```diff
@@ -213,10 +213,11 @@ Use the below command to replace or update the secret:
 | skipCertificateValidation | A boolean that enables/disables certificate validation of the csm-authorization proxy server. | No | true |
 
 
-10. Install the driver using `csi-install.sh` bash script by running `cd dell-csi-helm-installer && ./csi-install.sh --namespace vxflexos --values myvalues.yaml`. You may modify the release name with the `--release` arg. If arg is not provided, release will be named `vxflexos` by default. 
+10. Install the driver using `csi-install.sh` bash script by running `cd dell-csi-helm-installer && ./csi-install.sh --namespace vxflexos --values myvalues.yaml --helm-charts-version csi-vxflexos-2.10.1`. You may modify the release name with the `--release` arg. If arg is not provided, release will be named `vxflexos` by default. 
 Alternatively, to do a helm install solely with Helm charts (without shell scripts), refer to `helm/README.md`.
 
  *NOTE:*
+- If you do not specify the `--helm-charts-version` flag, by default the `csi-install.sh` script will clone the version of the helm chart that is specified in the driver's [csi-install.sh](https://github.com/dell/csi-powerflex/blob/main/dell-csi-helm-installer/csi-install.sh#L24) file. If you wish to install the driver using a different version of the helm chart, you need to include this flag. Also, remember to delete the `helm-charts` repository present in the `csi-powerflex` directory if it was cloned before.
```

Comment:
Similar to comments above

Reply:
Hi @gallacher Thanks for the review.
Since we are not releasing the driver i.e. v2.10.1 so customer will clone/checkout the v2.10.0 branch that is the latest release of driver and will try to run the install script and by default it will clone the same version of helm chart that is matching with the driver version so here in this case we need to pass this flag to override the default value.
Earlier we have never mentioned this flag before so in case if customer is trying out with the older command then they will end up with installing the driver with older version of helm-chart that has the bug.
That's why we thought of introducing this Note point here to avoid any confusion.

Reply:
Hi @gallacher, adding to Adarsh's comment, during driver installation, the helm-charts repository is cloned with a specified branch (Ref: https://github.com/dell/csi-powerflex/blob/main/dell-csi-helm-installer/csi-install.sh#L384).
Since we are not releasing the driver, the branch `csi-vxflexos-2.10.0` will be cloned by default which is defined in the `csi-install.sh` script here https://github.com/dell/csi-powerflex/blob/main/dell-csi-helm-installer/csi-install.sh#L24 if we don't specify that flag explicitly.

Reply:
@adarsh-dell / @suryagupta4 - is this acceptable from a customer perspective? Please work with @sharmilarama and team to ensure we are aligned properly.

Reply:
If that option is available, then there shouldn't be an issue with utilizing it, as this specific flag was introduced precisely to handle such scenarios.